src/js/ripple/remote.js

```diff
@@ -1424,10 +1425,19 @@ Remote.prototype.requestAccountTx = function(options, callback) {
   }, options);
 
   request.once('success', function(res) {
-    if (options.parseBinary) {
-      res.transactions = res.transactions.map(Remote.parseBinaryTransaction);
+    if (!options.parseBinary) {
+      request.emit('transactions', res);
+      return;
     }
-    request.emit('transactions', res);
+
+    async.mapSeries(res.transactions, function(transaction, next) {
+      async.setImmediate(function() {
```

Comment:
Ah. I didn't read the docs, I read [the code](https://github.com/caolan/async/blob/master/lib/async.js#L86-L113). Internally async seems to [prefer](https://github.com/caolan/async/blob/master/lib/async.js#L553) setImmediate. I also prefer to use setImmediate. I'll see if I can get it documented on the async readme so we can use it instead. Leave this pull request open until then